README.md

```diff
@@ -1 +1,8 @@
-# DesignPatterns
\ No newline at end of file
+
+List of patterns:
+
+1)Factory Pattern
+
+2)Decorator Pattern
```

Comment:
1. How is Mobile Decorator Different from Mobile App Decorator? Why do you need inheritance if you have just a single class? 
2. If mobile is an interface, how are the sub classes inheriting from it? 
3. Why is payment app a function of mobile?
4. Why does the decorator have a list of apps? 
5. Why is paymentApp a function of MobileAppDecorator?